.rspec

```diff
@@ -1 +1,3 @@
 --require spec_helper
+--color
```

Comment:
Yeah I could have put it in another PR or on master but I figured it was fine here